### DISCUSSION TITLE
SummaryInformation element does not have Comments property

### DISCUSSION THREAD

**Discussion by stefanhubertus** [wixtoolset/issues#7984] [Questions]
I have tried to add the Comments property to a SummaryInformation element in WiX 4.0.3. According to this documentation, the property should be available: https://wixtoolset.org/docs/schema/wxs/summaryinformation/

Unfortunately I always get this error message when trying to build (or as a preview in VS 2022): error WIX0004: The SummaryInformation element contains an unexpected attribute 'Comments'.

Is there something wrong with the SummaryInformation element or do I overlook a requirement here? The other attributes are offered correctly.

**Comment by barnson** [comment]
It's supported in WiX v5.

**Reply by stefanhubertus** [reply]
I am not aware off that this version already exists. So is the documentation wrong and I now have to go back to WiX v3 to get this feature back? Or is there a workaround somebody could suggest?

**Comment by SylwesterZarebski** [comment]
Why this was not backported to newest v4? When v5 will be released, when v3 has been given deadline?

**Reply by robmen** [reply]
It wasn't backported because it was not a Customer priority.

You can follow the v5 release timeline via WiX Meeting highlights on the FireGiant blog. [This is the latest](https://www.firegiant.com/blog/2024/2/27/wix-online-meeting-277-highlights/).

**Reply by SylwesterZarebski** [reply]
Thanks for informations, i'm waiting impatiently for v5.

**Reply by robmen** [reply]
Note: If you don't want to wait for such things, you could [become a Customer](https://www.firegiant.com/services/support-programs/).
